guide/glossary.md

```diff
@@ -241,46 +294,39 @@ A small piece of data that has all the information needed to generate a specific
    height = 400
 %}
 
-## Private key
-Every bitcoin address has a public key and a corresponding private key, together they are called a keypair. If you have access to both the public and private key, you effectively control the funds in the address. As with HD Wallets there are also keypairs that control *branches* in the hierarchical tree of the wallet, and at the very top is the extended keypair (x-pub and x-prv for short) that control all the addresses in the wallet.
-
-The private key is a 64 hexadecimal (or 256 if described in binary 1’s and 0’s) character string generated by the encryption algorithm. They look something like this in hexadecimal form:
-
-`5KYZdUEo39z3FPrtuX2QbbwGnNP5zTd7yyr2SC1j299sBCnWjss`
+### Recovery phrase
 
-Or for the extended private key:
+_Also referred to as Seed, Mnemonic, and Backup phrase._
 
-`xprv9zrji5mK3nb4RbuR2ZYFtyzK3gn78KnEzkNP4ZxwwPPwcgQQVZqnjTMAGxmmM3jpmfsthQUtfD9iYPvnaqwejCjcyEswLqEhX4LPKNFUXT5`
+The controlling keypair of a bitcoin wallet can be derived from a *recovery phrase* of 12 words (or 18 or 24, which is less common) from a standardized list, defined in [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki). The recovery phrase provides full access to a bitcoin wallet as it contains the private key and is therefore very valuable. It’s extremely important to keep it safe, both from other people getting access to it and for yourself not to lose it by creating one or several backups of the phrase. In many applications most of this work falls on the user and it’s important to acknowledge the responsibility here of the makers of the application to ensure that the user is able and aware of how to securely store a recovery phrase backup.
 
-## Public key
-A bitcoin address' public key can be derived from the private key. The address itself is a hash of the public key.
+Many wallet-applications work with HD Wallets and recovery phrases, and are interoperable, meaning you can change the application that can control your wallet should you wish (although there are some caveats depending on if they support just BIP32 or also BIP44).
 
-## Replace-by-fee (RBF)
-A node policy that allows an unconfirmed transaction to be replaced with a different transaction that spends at least one of the same inputs and which pays a higher transaction fee. This can sometimes be useful if a transaction has got *stuck* during times of higher-than-normal network-fees.
+### Remote procedure calls (RPC)
 
-## Recovery phrase
+A remote procedure calls (RPC) is when a command is sent to a remote system where it's processed and results are returned. Bitcoin uses RPC to connect to running instances of bitcoind and issue commands such as sending transactions or querying the Bitcoin block chain.
```

Comment:
```suggestion
A remote procedure calls (RPC) is when a command is sent to a remote system where it's processed and results are returned. The Bitcoin Core application uses RPC to connect to running instances of bitcoind and issue commands such as sending transactions or querying the Bitcoin block chain.
```
Good to clarify that the Bitcoin Core implementation of Bitcoin uses this (and not somehow Bitcoin in general).

Reply:
Good call, added